src/components/activity/ActivityDirectivesTable.svelte

```diff
@@ -138,13 +174,36 @@
     return activityDirective.id;
   }
 
+  function getSelectedActivityDirectives(): ActivityDirective[] {
+    const directives: ActivityDirective[] = [];
+    bulkSelectedActivityDirectiveIds.forEach(id => {
+      const found = activityDirectives.find(item => item.id === id);
+      if (found !== null && found !== undefined) {
+        directives.push(found);
+      }
+    });
+    return directives;
+  }
+
   function scrollTimelineToActivityDirective() {
     const directiveId = bulkSelectedActivityDirectiveIds.length > 0 && bulkSelectedActivityDirectiveIds[0];
     const directive = activityDirectives.find(item => item.id === directiveId) ?? null;
     if (directive?.start_time_ms !== undefined && directive?.start_time_ms !== null) {
       dispatch('scrollTimelineToTime', directive.start_time_ms);
     }
   }
+
+  function copyActivityDirectives({ detail: activities }: CustomEvent<ActivityDirective[]>) {
+    plan !== null && copyActivityDirectivesToClipboard(plan, activities);
```

Comment:
Can we use an `if` conditional here instead of `&&` to make it a little easier to glance at? It currently looks as if it should return boolean.